src/modules/home/components/TrendingList.tsx

```diff
@@ -0,0 +1,19 @@
+import { Link } from "react-router-dom";
+import { ITrendingMangaApp } from "../../common/types/app-types";
+
+const TrendingList = ({ slug, title, coverImage }: ITrendingMangaApp) => {
+    return (
+        <div className="card w-full bg-base-100 shadow-xl image-full carousel-item h-full">
+            <figure className="w-full rounded-none"><img src={coverImage} alt={`Trending Manga: ${title}`} className="rounded-none w-full object-top" /></figure>
```

Comment:
`Line 7`: I am a bit curious about the `rounded-none`, is there a reason you have to deliberately set the border radius to 0px? Is an element's border-radius not normally unset/none by default?

Reply:
Unfortunately a lot of daisyUI components come with rounded boarders so i have to overwrite that style myself. This is one instance of that